runelite-api/src/main/java/net/runelite/api/Varbits.java

```diff
@@ -297,6 +297,13 @@
 	PERSONAL_POINTS(5422),
 	RAID_PARTY_SIZE(5424),
 
+	/**
+	 * Braziers
```

Comment:
Description should probably refer to them being the My Arm ones, since Braziers also exist in Wintertodt.